assets/css/images/sensei-admin-menu.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="349" height="349" viewBox="0 0 349 349" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't see this file being used anywhere as we directly encoded the svg element in the background css property. Why don't we use `url('./images/sensei-admin-menu.svg')` instead?

Reply:
I had to manipulate the original SVG to get the colors to match the ones in the sidebar, so although the `assets/css/images/sensei-admin-menu.svg` is not included in the build, I'm committing it in case we need to manipulate it again in the future and convert it to a data URI again.

Also, I'm not linking to it directly to avoid the extra HTTP request (which would happen often since the icon is needed throughout WP admin).

Reply:
Nice! I agree that the inline svg is better but I was confused why we needed both the file and encoded one. So if this file needed in case we need to do more modifications on it in the future, can we add a comment on it clarifying it?

Reply:
Surely! https://github.com/Automattic/sensei/pull/4310/commits/3886c3f2259e27eb9caeb36787df21c9aee541cf